subprojects/packagefiles/libkqueue/meson.build

```diff
@@ -138,8 +124,9 @@ else
 endif
 
 configure_file(
-    input : 'config.h.meson',
+    input : 'config.h.in',
     output : 'config.h',
+    format : 'cmake',
```

Comment:
I believe there had been a reason why I opted to chose to copy from upstream instead of using the cmake format, might have been that meson does not play well with `cmakedefine01` ?

I'll check on this as soon as I find time.

Reply:
I found no problem and the issue I've had in the past was issues with getting meson to substitute the `@VAR@`s within version.h

The only issue I did encounter was a problem when compiling with MinGW, but you've already fixed that one https://github.com/mheily/libkqueue/commit/f475d065c9a6a99f8a6c38e53abe0acebe184735